submariner/OVN-Interconnect.md

```diff
@@ -0,0 +1,126 @@
+# Submariner Enhancement for OVN Interconnect
+
+<!-- Add link to issue/epic if available -->
+
+## Summary
+
+OVN CNI is moving to a new network topology powered by OVN Interconnect Feature. This allows independent
+OVN deployments to be interconnected by OVN-managed GENEVE tunnels. Submariner does not work with these changes
+out of the box and this proposal explains the changes required to support it.
+
+## Proposal
+
+With OVN Interconnect we can have two types of deployment
+
+### Single Zone(Global Zone)
+
+A single-zone deployment will have only one OVN database and a set of master nodes programming it. This is similar to the topology that we
+use now. But it will have a single `global` zone assigned to it with a transit switch which is not involved in any packet forwarding. Though this setup is expected to work by default, the subnet that the transit switch
+used overlaps with the subnet Submariner uses for the logical router. So we need to agree upon a non-overlapping subnet for Submariner.
+
+### Multiple Zone
+
+In a multiple-zone setup, we will have an OVN database and a set of master nodes for each zone. Transit switches connect the zones. The OVN-Kubernetes services ensure that the necessary routes are added for pod and service reachability across nodes in different zones.
```

Comment:
How do we know if its multi zones ovn or single one ?

Reply:
We will have to scan all the nodes to know if there is multiple node configured.

Reply:
Is a one-time test enough?
I mean, is it possible that at t0 we have a single zone and after some time new zones(s) will be added?